cmd/kubeadm/app/images/images.go

```diff
@@ -39,10 +40,11 @@ func GetCoreImage(image string, cfg *kubeadmapi.MasterConfiguration, overrideIma
 	}
 	repoPrefix := kubeadmapi.GlobalEnvParams.RepositoryPrefix
 	return map[string]string{
-		KubeEtcdImage:              fmt.Sprintf("%s/%s-%s:%s", repoPrefix, "etcd", runtime.GOARCH, etcdVersion),
+		KubeEtcdImage:              fmt.Sprintf("quay.io/coreos/etcd:%s", cfg.Etcd.Cluster.Version),
```

Comment:
The `cfg.Etcd.Cluster.Version` val can be set in the config file. If nothing is provided, a default is set [here](https://github.com/jamiehannaford/kubernetes/blob/ba18497cdd96facc62060ff199a3b454c5ac5e17/cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go#L83). I didn't think there would be much need for a user to override the operator image tag. Do you think that's a conceivable use case? I didn't want to bloat the config unnecessarily.